File: C++/uri_1069_diamantes.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int remove_diamantes(string mina){
    int esquerda = 0, diamantes = 0;
    for (int i = 0; i < mina.length(); i++){
        if (mina[i] == '<'){
            esquerda = true;
        }
        else{
            esquerda = false;
        }
        if (esquerda == true){
            for (int d = i; d < mina.length(); d++){
                if (mina[d] == '>'){
                    mina[i] = 'w';
                    mina[d] = mina[i];
                    diamantes += 1;
                    esquerda = false;
                    break;
                }
            }
        }
    }
    return diamantes;
}


int main(){
    int cases;
    char mina[1000];
    cin >> cases;
    for (int i=0; i<cases; i++){
        cin >> mina;
        cout << remove_diamantes(mina) << endl;
    }

    return 0;
}

File: URI/uri_1069_diamantes.py
def main():
    for c in range(0, int(input())):
        mina = list(input())
        print(remove_diamante(mina))


def remove_diamante(mina):
    esquerda = 0
    diamantes = 0
    for c in range(0, len(mina)):
        if mina[c] == '<':
            esquerda = True
        else:
            esquerda = False
        if esquerda == True:
            for d in range(c, len(mina)):
                if mina[d] == '>':
                    mina[d] = mina[c]= 'retirado'
                    diamantes += 1
                    esquerda = False
                    break
    return diamantes


main()
